cli/cli_kafka_runner_test.go

```diff
@@ -0,0 +1,343 @@
+package cli
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/featurebasedb/featurebase/v3/cli/internal"
+	"github.com/featurebasedb/featurebase/v3/cli/kafka"
+	"github.com/featurebasedb/featurebase/v3/errors"
+	"github.com/featurebasedb/featurebase/v3/logger"
+	"github.com/stretchr/testify/require"
+)
+
+// These variables are used throught tests to refer to resouces needed for the
+// tests (i.e. where to find the kafka service)
+var (
+	pilosaHost     string
+	pilosaTLSHost  string
+	pilosaGrpcHost string
+	kafkaHost      string
+	registryHost   string
+	certPath       string
+)
```

Comment:
Can we include the default values here instead of in a bunch of `if/else` blocks below?
For example, 
```
pilosaHost = "pilosa:10101"
```
based on the current logic below.

With that said, I think the default should be the local value, and running in docker or CI should set up with configuration or environment variables.